blocks/stopwatch/src/dev.tsx

```diff
@@ -2,17 +2,37 @@
  * This is the entry point for developing and debugging.
```

Comment:
I'm not seeing the `README.md`s in the diff, have you checked the descriptions are still up to date? I have a feeling they're not, and might be kind of annoying to update 🙃 

They unfortunately appear pretty prominently on the Hub

Reply:
I have not touched the READMEs at all. Will have a quick look

Reply:
Updated READMEs except for timer which will do tomorrow. Put a note saying it needs updating instead